rq/results.py

```diff
@@ -181,7 +181,11 @@ def serialize(self):
         if self.exc_string is not None:
             data['exc_string'] = b64encode(zlib.compress(self.exc_string.encode())).decode()
 
-        serialized = self.serializer.dumps(self.return_value)
+        try:
+            serialized = self.serializer.dumps(self.return_value)
+        except:  # noqa
+            return data
```

Comment:
Mind putting a simple `<Unserializable return value>` string as the `return_value`? This tells the user that the job actually returns something that we can't keep.

Reply:
Oh yes sure! I'll put the same string as in the `job._result` code

Reply:
Done in https://github.com/rq/rq/pull/1888/commits/5bf8239ea2a8378a6d0e344fcf80aaa07bcb37f3
I took the liberty of creating a default value so we don't repeat the same string twice in the code - I think it's cleaner this way